src/components/UTIL_PageHeader.component

```diff
@@ -63,17 +63,17 @@
     <apex:attribute name="saveAndNewDisabled" type="Boolean" default="false"
         description="Whether the save and new button should be disabled, defaults to false."/>
 
-    <div class="slds-page-header" role="banner">
+    <div class="slds-page-header" role="banner" id="header">
```

Comment:
Should this have a more specific Id value to avoid a conflict? For example, `npsp-page-header`